### PR TITLE
Some more link fixes

### PR DESCRIPTION
**Pull request by smathy**

### CODE REVIEW

File: source/doc-pages/learn/adapters.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Adapters
 
 * [how-to](/learn/adapters/how-to) - learn how to build your own adapter
-* [rom-sql](/learn/adapters/sql)
-* [rom-cassandra](/learn/adapters/cassandra)
+* [rom-sql](/learn/adapters/adapter/sql/)
+* [rom-cassandra](/learn/adapters/adapter/cassandra/)
```
Comment (solnic) [inline source/doc-pages/learn/adapters.md]: this is not right, could you please move sql & cassandra to adapters dir? my bad...

Comment (smathy) [inline source/doc-pages/learn/adapters.md]: Done in e3a1d8c

**Comment by solnic** [comment]
Thanks man

**Comment by smathy** [comment]
:+1: thank *you*.

**Pull request merged**